R/type.R

```diff
@@ -62,8 +62,11 @@ xportr_type <- function(.df,
 
   ## Pull out correct metadata
   metadata <- metadata %||%
-    attr(.df, "_xportr.df_metadata_") %||%
+    attr(.df, "_xportr.df_metadata_")
```

Comment:
Why the change here? Was it a matter of being more readable?

I think this implementation is also on other functions

Reply:
I reverted this back to how it was in `devel`